content/docs/09.administrator-guide/02.deployment/04.restack.md

```diff
@@ -1,56 +1,197 @@
+2. You have an active Azure subscription
+3. You have an active Azure user with the "Owner" role.
+
+
+Follow these steps to create a new Azure App Registration in order to authenticate your Azure account with Restack and deploy Azure resources from Restack.
+
+1. Create a new App Registration in your Azure Active Directory 
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/6b355dd2401ab94754258f4fe12ff51c52c6f0bc-3020x1652.png)
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/dc0fa6c05a65e60d9ff7c323b25b5ea116e4e7e1-3024x1650.png)
+
+2. Copy the Application (client) ID and paste it into the Restack UI setup as the Client ID.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/392b15df09de170e1bb7632880f42ff093852853-2564x1730.png)
+
+3. Copy the Directory (tenant) ID and paste it in Restack UI as the Tenant ID.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/cb0e76db65825a3d097b764735e6de29d68195ff-2590x1728.png?w=450)
+
+4. For your newly created App Registration, add an API Permission called "user_impersonation".
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/d3c38be874bce1bfe63c60a1bd6a2da75b77cce5-3024x1648.png)
+
+5. For your App Registration, go to "Certificates & secrets" and create a new client secret. Copy the value of the secret and paste it in Restack UI as the Client Secret.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/2cc097fd4e153bab779de038f153a83bb2a38c59-3024x1648.png)
+
+6. Go to your "Subscriptions", and choose the subscription you want to use for Restack. In the "Access Control (IAM)" section, access "Role assignments" and add a new role assignment. Choose the "Owner" role (Privileged administrator roles) & Azure Kubernetes Service RBAC Cluster Admin (Job function roles) and click "Next".
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/6451cf86e77b7fb886d63578734858fe00c8f149-3024x1650.png)
+![](https://cdn.sanity.io/images/ev3amoz3/production/1121d4525b7659406f38ddcab0e745ff1d84cb1f-3024x1650.png)
+
+7. Choose "User, group or service principal" in "Assign access to" and click on "Select members". In the opened search bar type in "restack". The "restack" App Registration you've created should pop up. Select it and click on "Next". Lastly click on "Review + assign". Now in "Role assignments" section you should see the "restack" app
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/3a8e74393fc5b3ee3b0598ef69475b9e4d738891-3024x1650.png)
+
+8. Go to "Subscriptions" and copy the Subscription ID to paste in Restack UI.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/dfaefa1f878e41dce37339de365cb6e32c88ea9d-2564x1726.png)
+
+9. Click on "Connect cloud". 
+
+If the cloud account was added successfully and Restack can connect to it, you can proceed to create a cluster and provision applications.
+
+## One-click cluster creation with Restack
 
-Once you have added your credentials:
 1. Navigate to the **Clusters** tab on the same page and click on **Create cluster**.
```

Comment:
```suggestion
1. Navigate to the **Clusters** tab and click on **Create cluster**.
```

Given that the previous setup was separate per cloud, it's unclear what the same page means so worth cutting or clarifying.